sections/ads.txt

```diff
@@ -2,6 +2,8 @@
 !-------------- Ads --------------------------------------------------------!
 !---------------------------------------------------------------------------!
 !
+/bannerek/*
+hu##[class*="banner"]
```

Comment:
*= is slow selector, and the rule is too general

Reply:
#264